### DISCUSSION TITLE
[Base]  Including OSM attributes

### DISCUSSION THREAD

**Discussion by HurdyGurdy123** [OvertureMaps/data#179] [Discussion]
### Category Feedback

Raw OSM includes a huge amount of attribution some  of it in the form of tags for example power line might include

`{    "cables": "6",  "circuits": "2",   "frequency": "50","power": "line",  "voltage": "132000",    "wires": "single"​ }​`

You can obtain this from other sources such as [https://overpass-turbo.eu/](https://overpass-turbo.eu/)

Transportation > roads and Buildings have multiple useful attributes

Are there any plans to include greater attribution in other themes notably infostructure category?

 

### Dependency with other categories, if any.

No Dependency

**Comment by jenningsanderson** [comment]
Thanks for the question, @HurdyGurdy123. 

The `base` theme is intended to host features that are required to render a complete base map that do not exist in other Overture themes.

As you noticed, the `base` theme does not have a fully developed schema in the same way that `buildings` or `places` has. Instead, we have a `source_tags` column that contains additional attributes.

In the future, some of these feature types may graduate to their own theme, such as a `water` or `landuse` theme. 

For specific details about things like power transmission lines, however, OSM is going to contain richer information that we can build a specific schema around. For this reason, we make it easy to join Overture features back to OSM features by including the OSM type, id, and version on every Overture feature derived from OSM.
